README.md

```diff
@@ -11,6 +11,20 @@ A Visual Studio Code extension that listens to twitch chat and allows viewers to
 
 ![](./resources/screenshot-example.gif)
 
+<a name="quickstart"></a>
+## Quickstart Guide
+
+
+Install [VS Code](https://code.visualstudio.com/Download) if you don't already have it.
+
+Install [Twitch Themer](https://marketplace.visualstudio.com/items?itemName=MichaelJolley.vscode-twitch-themer) from the Visual Studio Marketplace
+Connect Twitch Themer to Twitch from the [Command Palette](#command-palette) (Ctrl + Shift + P)
+
+This opens the Twitch Authentication page. Login to your Twitch account
+
+Done!
+
+<a name="twitch-chat-commands"></a>
```

Comment:
I think we can remove all these added anchors. By default, GitHub turns the <h> tags into anchors.

Reply:
Done